src/pynwb/validate.py

```diff
@@ -107,6 +109,12 @@ def _get_cached_namespaces_to_validate(path: str) -> Tuple[List[str], BuildManag
         "doc": "Whether or not to print messages to stdout.",
         "default": False,
     },
+    {
+        "name": "driver",
+        "type": list,
```

Comment:
shouldn't "driver" be a string, not a list?

Reply:
yes